GFW/convert.py

```diff
@@ -0,0 +1,103 @@
+import os
+import time
+import xarray as xr
+import rioxarray as rxr
+from rasterio.enums import Resampling
+from rasterio import Affine
+from urllib.request import urlretrieve
+from osgeo import gdal, osr
+import numpy as np
+import cftime as cf
+import datetime
+
+# vars
+sdate = datetime.datetime(2020, 1, 1)
+edate = datetime.datetime(2050, 1, 1)
```

Comment:
I am not familiar with this product, it is strange for me to think of a reference product whose end date is future. Is this really the end?

Reply:
What time bounds would you usually expect from a ref product? Also, I got these numbers from the paper! Here: "Throughout, we focused on the first thirty years of natural forest regrowth, because 2020 to 2050 represents a biophysically critical and policy-relevant window for reaching net zero emissions and limiting the most negative effects of global warming." And in reference to measurements they used, they added "given our 2020 to 2050 focus." So, I took that to mean that the applicable comparison period would be 2020 to 2050? But I 100% could have misunderstood